### PR TITLE
Issue 7: public board

### PR DESCRIPTION
**Pull request by lpatmo**
TODO: 
[X] Create the heart/toggles for each card
[X] Filter the cards for each column by category
[X] Schema updates to define what is in `preferences`, and make sure `createdAt` is included
[X] Renamed `cards` to `entries`
[X] Installed `@material-ui/icon` library

### CODE REVIEW

File: imports/api/entries/entries.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  'tz.daylightSaving': String,
+  lookingFor: String,
+  oneLineIntro: String,
+  verified: Boolean,
+  active: Boolean,
+  preferences: {
+    type: Array,
+    optional: true,
+  },
+  'preferences.$': {
```
Comment (lpatmo) [inline imports/api/entries/entries.js:32]: What is `preferences.$` meant for?

File: imports/api/entries/server/publications.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Entries } from '../../lib/collections/entries';
+import Entries from '/imports/api/entries/entries.js';
 
 Meteor.publish('entries.board', function(argument) {
```
Comment (lpatmo) [inline imports/api/entries/server/publications.js:3]: Can we increase to more than 19, here? (Wondering if we want "scroll down for more entries", here)

File: imports/api/entries/entries.js (hunk 1 above)
Comment (gauravchl) [reply imports/api/entries/entries.js:32]: The dollar sign? It indicates the single item in an array. https://github.com/aldeed/simple-schema-js#more-shorthand

But i'm not sure the use case of preferences field though.

File: imports/api/entries/server/publications.js (hunk 2 above)
Comment (gauravchl) [reply imports/api/entries/server/publications.js:3]: Yeah why not, but for scrolling behaviour we should consider dynamic loading

File: imports/api/entries/entries.js (hunk 1 above)
Comment (lpatmo) [reply imports/api/entries/entries.js:32]: OHH I understand now, thanks!

File: imports/api/entries/server/publications.js (hunk 2 above)
Comment (lpatmo) [reply imports/api/entries/server/publications.js:3]: ++

**Pull request merged**